invisible_cities/reco/xy_algorithm_test.py

```diff
@@ -174,23 +285,21 @@ def test_discard_sipms(toy_sipm_signal_and_inds):
 
 
 def test_get_nearby_sipm_inds():
-    xs  = np.array([0,1,2,3,4,0,1,2,3,4,0,1,2,3,4])
-    ys  = np.array([0,0,0,1,1,1,2,2,2,3,3,3,4,4,4])
+    xs  = np.array([0, 1, 2, 3, 4, 0, 1, 2, 3, 4, 0, 1, 2, 3, 4])
+    ys  = np.array([0, 0, 0, 1, 1, 1, 2, 2, 2, 3, 3, 3, 4, 4, 4])
     pos = np.stack((xs, ys), axis=1)
-    qs  = np.ones (100) * 10 * units.pes
     xc, yc = (2, 2)
     c = np.array((xc, yc))
     d  = 1.5
-    sis = get_nearby_sipm_inds(c, d, pos, qs)
+    sis = get_nearby_sipm_inds(c, d, pos)
     for i in range(len(xs)):
-        if i in sis:
-            assert np.sqrt((xs[i] - xc)**2 + (ys[i] - yc)**2) <= d
-        else:
-            assert np.sqrt((xs[i] - xc)**2 + (ys[i] - yc)**2) >  d
+        if i in sis: assert np.sqrt((xs[i] - xc)**2 + (ys[i] - yc)**2) <= d
+        else       : assert np.sqrt((xs[i] - xc)**2 + (ys[i] - yc)**2) >  d
 
 
+@mark.skip("Functions removed")
```

Comment:
Why do you not delete this test at all?

Reply:
It will be. There are multiple skipped tests in the code and I am thinking of a PR that removes all of them. I don't mind removing it here, directly.

Reply:
Ah, ok, if there's another PR in sight, there's no need to remove it here.